cmd/notary-server/config.go

```diff
@@ -278,3 +282,31 @@ func parseServerConfig(configFilePath string, hRegister healthRegister) (context
 		ConsistentCacheControlConfig: consistentCache,
 	}, nil
 }
+
+func setupSignalTrap() {
+	c := make(chan os.Signal, 1)
+	signal.Notify(c, notary.NotarySupportedSignals...)
+	go func() {
+		for {
+			signalHandle(<-c)
+		}
+	}()
+}
+
+// signalHandle will increase/descrease the logging level via the signal we get.
+func signalHandle(sig os.Signal) {
+	switch sig {
+	case syscall.SIGUSR1:
+		if err := utils.AdjustLogLevel(true); err != nil {
+			fmt.Printf("Attempt to increase log level failed, will remain at %s level, error: %s", logrus.GetLevel(), err)
```

Comment:
The reason why I don't use `logrus` here and blow is  from here:
https://github.com/docker/notary/pull/748#discussion-diff-63277219R279